modules/installing-private-image-registry-private-azure.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+//* registry/configuring_registry_storage-azure.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-private-image-registry-private-azure"]
+= Optional: Preparing a private Microsoft Azure cluster for a private image registry
+
+By installing a private image registry on a private Microsoft Azure cluster, you can create private endpoints. Private endpoints disable public facing endpoints to the internal registry, adding an extra layer of security to your {product-title} deployment.
+
+Use the following guide to prepare your private Microsoft Azure cluster for installation with a private image registry.
+
+.Prerequisites
+
+* You have created a global Microsoft Azure account to use with the following permissions:
++
+.Required permissions for creating a private image registry endpoint
```

Comment:
@flavianmissi @wewang58  when using azure workload identity, those permissions should also be required on identity for image-registry, is that right?

Reply:
Set to "Required"